go/libkb/sig_chain.go

```diff
@@ -370,6 +381,29 @@ func (sc *SigChain) VerifyChain(m MetaContext) (err error) {
 		curr.chainVerified = true
 	}
 
+	newHPrevInfo := NewInitialHPrevInfo()
+	for i := 0; i < len(sc.chainLinks); i++ {
+		curr := sc.chainLinks[i]
+
+		if hPrevInfo := curr.GetHPrevInfo(); hPrevInfo != nil {
```

Comment:
Can you provide a comment about it being an error if null, at some point?  How will we enforce that? I.e., how will we know when to error if an hprev is expected but isn't there?

Reply:
This function will have to play old links too, right? Would we need to do something like set a feature-flag and reject the link if the feature-flag is on, it doesn't have an hprev_info and it's ctime is after some "enforce hprev_infos" day?